doc/voq/aggregate_voq_counters.md

```diff
@@ -0,0 +1,131 @@
+```
+
+From linecard - cmp217-5 for asic1 (existing CLI)
+```
+admin@cmp217-5:~$ show queue counters -n asic1 "cmp217-5|asic1|Ethernet256" --voq
+For namespace asic1:
+                      Port    Voq    Counter/pkts    Counter/bytes    Drop/pkts    Drop/bytes    Credit-WD-Del/pkts
+--------------------------  -----  --------------  ---------------  -----------  ------------  --------------------
+cmp217-5|asic1|Ethernet256   VOQ0            1111            55550            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ1              45             2250            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ2               9              450            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ3              91             4550            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ4              55             2750            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ5              88             4400            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ6              21             1050            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ7              48            14437            0             0                     0
+
+```
+
+From supervisor (same command extended for sup.)
+
+```
+admin@cmp217:~$ show queue counters "cmp217-5|asic1|Ethernet256" --voq
+                      Port    Voq    Counter/pkts    Counter/bytes    Drop/pkts    Drop/bytes    Credit-WD-Del/pkts
+--------------------------  -----  --------------  ---------------  -----------  ------------  --------------------
+cmp217-5|asic1|Ethernet256   VOQ0            1234            61700            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ1              57             2850            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ2            1009            50450            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ3             547            27350            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ4             266            13300            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ5             133             6650            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ6              45             2250            0             0                     0
+cmp217-5|asic1|Ethernet256   VOQ7              52            15809            0             0                     0
+
+```
+
+### Testing Requirements/Design  
+#### System Test cases
+Send traffic across different ASICs and ensure aggregate counters are correctly displayed.
+
+### Limitations and future work
+1. Currently we are not exposing redis instance over midplane IP for single ASIC linecards as redis runs in protected mode.
```

Comment:
How will this feature work for system having mix of J2C+ as well as J2 based line cards? Since some of J2 cards are single asic, are you saying those counters won't be part of counter aggregation on SUP?